gnosis_vpn-lib/src/config/v3.rs

```diff
@@ -337,9 +378,39 @@ impl Default for options::Options {
 impl From<BufferOptions> for options::BufferSizes {
     fn from(buffer: BufferOptions) -> Self {
         options::BufferSizes::new(
-            buffer.bridge.unwrap_or("0 B".to_string()),
-            buffer.ping.unwrap_or("0 B".to_string()),
-            buffer.main.unwrap_or("5 MB".to_string()),
+            buffer.bridge.unwrap_or(Connection::default_bridge_buffer_size()),
+            buffer.ping.unwrap_or(Connection::default_ping_buffer_size()),
+            buffer.main.unwrap_or(Connection::default_main_buffer_size()),
+        )
+    }
+}
+
+impl Default for options::BufferSizes {
+    fn default() -> Self {
+        options::BufferSizes::new(
+            Connection::default_bridge_buffer_size(),
+            Connection::default_ping_buffer_size(),
+            Connection::default_main_buffer_size(),
+        )
+    }
+}
+
+impl From<MaxSurbUpstreamOptions> for options::MaxSurbUpstream {
+    fn from(surbs: MaxSurbUpstreamOptions) -> Self {
+        options::MaxSurbUpstream::new(
+            surbs.bridge.unwrap_or(Connection::default_bridge_max_surb_upstream()),
+            surbs.ping.unwrap_or(Connection::default_ping_max_surb_upstream()),
+            surbs.main.unwrap_or(Connection::default_main_max_surb_upstream()),
+        )
+    }
+}
+
+impl Default for options::MaxSurbUpstream {
+    fn default() -> Self {
+        options::MaxSurbUpstream::new(
+            Connection::default_bridge_max_surb_upstream(),
+            Connection::default_ping_max_surb_upstream(),
+            Connection::default_main_max_surb_upstream(),
```

Comment:
You could use the https://docs.rs/smart-default/latest/smart_default/derive.SmartDefault.html